<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filters</class>
 <widget class="QDialog" name="Filters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gaussianButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Gaussiano</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_9">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="laplacianoButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Laplaciano</string>
   </property>
  </widget>
  <widget class="QPushButton" name="passaAltasButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Passa Altas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prewittHxButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Prewitt Hx</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prewittHyHxButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Prewitt Hy Hx</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sobelHxButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Sobel Hx</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sobelHyButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Sobel Hy </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>FILTROS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
